From 7caf9dae8b4e0eba767b552ebeae25c3c410eb8d Mon Sep 17 00:00:00 2001
From: Hai Pham <hai.pham.ud@renesas.com>
Date: Wed, 28 May 2025 18:24:14 +0700
Subject: [PATCH 52/52] configs: ironhide: Increase SYS_CBSIZE to 10KiB

Increase SYS_CBSIZE to 10KiB to accommodate long console input,
especially for CLI input in CI test.

Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 configs/r8a78000_ironhide_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/r8a78000_ironhide_defconfig b/configs/r8a78000_ironhide_defconfig
index f2c5c5dfa1c..61de7b6b39a 100644
--- a/configs/r8a78000_ironhide_defconfig
+++ b/configs/r8a78000_ironhide_defconfig
@@ -17,6 +17,7 @@ CONFIG_TARGET_IRONHIDE=y
 CONFIG_SYS_CLK_FREQ=1066666667
 CONFIG_SYS_BOOT_GET_CMDLINE=y
 CONFIG_SYS_BARGSIZE=2048
+CONFIG_SYS_CBSIZE=10240
 CONFIG_BOOTCOMMAND="tftp 0x61080000 Image && tftp 0x61000000 r8a78000-ironhide.dtb && booti 0x61080000 - 0x61000000"
 CONFIG_SYS_CBSIZE=2048
 CONFIG_CMD_MEMTEST=y
-- 
2.34.1

